src/sync/incomingSyncUtils.js

```diff
@@ -329,7 +329,7 @@ export function createOrUpdateRecord(database, settings, recordType, record) {
         itemBatch: itemBatch,
         snapshotNumberOfPacks: parseNumber(record.snapshot_qty) * packSize,
         packSize: 1, // Pack to one all mobile data
-        expiry: parseDate(record.expiry),
+        expiryDate: parseDate(record.expiry),
```

Comment:
Was this a bug? OutcomingSyncUtils has it correct 🤔

Reply:
yeah it was a bug